src/api/academy/request.ts

```diff
@@ -0,0 +1,22 @@
+import type { AcademyListItem } from "../../type/academy";
+import http from "../instance";
+
+// API 엔드포인트
+const ACADEMY_API = {
+  BASE: "/academies",
+};
+
+// API 함수들
+export const academyApi = {
+
+  // 로그인
+  getList: async (): Promise<AcademyListItem[]> => {
+    try {
+      return await http.get<AcademyListItem[]>({
+        url: ACADEMY_API.BASE,
+      });
+    } catch (error) {
+      throw new Error("조회 실패");
+    }
```

Comment:
여기서는 try/catch 문 안 쓰고 API 함수만 작성하시면 됩니다! `useQuery` 에러 핸들링은 `queryClient`의 `QueryCache` 라는 곳에서 하고 (개별 핸들링은 불가능..) `useMutation`은 `onError` 콜백에서 하면 돼요

둘 다 기본으로 toast를 띄우도록 해뒀는데 필요하다면 수정해서 쓰시면 될 것 같아요

```suggestion
    return await http.get<AcademyListItem[]>({
      url: ACADEMY_API.BASE,
    });
```